scripts/multisig.js

```diff
@@ -0,0 +1,140 @@
+const { program } = require('commander');
+const inquirer = require('inquirer');
+
+const { RECOVERY_METHOD_KINDS, TWO_FACTOR_AUTH_KINDS } = require('../constants');
+const {
+    deleteRecoveryMethod,
+    listRecoveryMethodsByAccountId,
+    updateRecoveryMethod,
+} = require('../db/methods/recovery_method');
+
+function is2faRecoveryMethod(kind) {
+    return Object.values(TWO_FACTOR_AUTH_KINDS).includes(kind);
+}
+
+function isAccountValid(accountId) {
+    if (accountId.length === 64) {
+        return true;
+    }
+
+    if (process.env.NEAR_WALLET_ENV.startsWith('testnet')) {
+        return accountId.endsWith('.testnet');
+    }
+
+    return accountId.endsWith('.near');
+}
+
+function isEmail(detail) {
+    return detail.includes('@');
```

Comment:
This is overly simplistic but in this context we technically only care that it's not a phone number. Looking for the `@` just makes a convenient, dead simple validation on the email input.